lnrpc/signrpc/signer.pb.go

```diff
@@ -1383,6 +1383,13 @@ type MuSig2SessionRequest struct {
 	// experimental RPC was already released. Some of those changes affect how the
 	// combined key and nonces are created.
 	Version MuSig2Version `protobuf:"varint,6,opt,name=version,proto3,enum=signrpc.MuSig2Version" json:"version,omitempty"`
+	// A set of pre generated secert local nonces to use in the musig2 session.
```

Comment:
```suggestion
	// A set of pre-generated secret local nonces to use in the musig2 session.
```

Reply:
So these are auto generated, so fixing just the proto comments fixed these in turn.